buildroot-external/package/rpi-rf-mod/Config.in

```diff
@@ -25,19 +25,19 @@ config BR2_PACKAGE_RPI_RF_MOD_DTS_TINKER
 	  For the ASUS Tinkerboard
 
 config BR2_PACKAGE_RPI_RF_MOD_DTS_ODROID_C4
-	bool "ODROID-C4"
+	bool "Odroid-C4"
```

Comment:
I've captialized these a while ago since that is how Hardkernel market their products now, see #2406.

Reply:
Ah ok. I thought this was a correction on the other end (in RM github). So I will fix it accordingly. Thx for the hint.